### DISCUSSION TITLE
text size

### DISCUSSION THREAD

**Discussion by decide-you** [prasanthrangan/hyprdots#1898] [Q&A] [answered]
I dont want to use scaling in monitor settings, its kinda blurry and the interface is too big. Is there a way to change text size all over the system? Something like text-scailing for sight-impaired? Or do i need to change it manually for waybar + gnome + qt 5 + qt 6? Can i configure it in my userprefs (ik i can do that with org.gnome.desktop.interface what about other parts)

**Comment by kRHYME7** [comment]
Unfortunately some of the font sizes are parsed from the common.conf file. So you Might need to change that there. Can  you point out which part you needed to upscale? I think Wayland handles it well but for x11 apps, Wayland can't solve it's scaling issues.

**Reply by decide-you** [reply]
> Can you point out which part you needed to upscale?

system notifications and application launcher (rofi)

**Reply by kRHYME7** [reply, marked as answer]
Add this for rofi scaling
![image](https://github.com/user-attachments/assets/816bb4ea-5097-4970-abdd-abe86bfd9d7b)

for dunst change the font size on the dedicated template dunst.conf. do not change the dunstrc as it is autogenerated 
![image](https://github.com/user-attachments/assets/428047a0-984c-4526-9512-b4e291f9f7ed)
![image](https://github.com/user-attachments/assets/f82379b8-0246-4238-99ed-f127eca5c5a8)


run ` Hyde reload ` to regnerate those stuff.
